### PR TITLE
Add geoMAD Dev OWS

### PR DESCRIPTION
**Pull request by JM-GA**
Adding web services for new geoMAD layers to dev.
The v4 notation will be removed when the current geomedian web service layers are decommissioned - I have added V4 tp the terriacube menu to distinguish the two geomedian versions for testing in Dev.

### CODE REVIEW

File: dev/terria/terria-cube-v8.json [hunk 1]
```diff
@@ -106,6 +106,90 @@
                             "type": "group",
                             "name": "DEA Surface Reflectance Calendar Year (Landsat)",
                             "members": [
+                                {
+                                    "type": "wms",
+                                    "name": "DEA GeoMAD (Landsat 8 & 9 OLI-TIRS) v4",
```
Comment (robbibt) [inline dev/terria/terria-cube-v8.json:111]: Assume we will drop this when we add the layers to prod DEA Maps? For now, nicer to put "v4" inside the brackets 

```suggestion
                                    "name": "DEA GeoMAD (Landsat 8 & 9 OLI-TIRS, version 4)",
```

File: dev/terria/terria-cube-v8.json [hunk 2]
```diff
@@ -106,6 +106,90 @@
...
+                                    },
+                                    "tileErrorHandlingOptions": {
+                                        "ignoreUnknownTileErrors": true
+                                    },
+                                    "dateFormat": "'Year: 'yyyy",
+                                    "id": "hTd67a"
+                                },
+                                {
+                                    "type": "wms",
+                                    "name": "DEA GeoMAD (Landsat 7 ETM+) v4",
```
Comment (robbibt) [inline dev/terria/terria-cube-v8.json:139]:
```suggestion
                                    "name": "DEA GeoMAD (Landsat 7 ETM+, version 4)",
```

File: dev/terria/terria-cube-v8.json [hunk 3]
```diff
@@ -106,6 +106,90 @@
...
+                                    },
+                                    "tileErrorHandlingOptions": {
+                                        "ignoreUnknownTileErrors": true
+                                    },
+                                    "dateFormat": "'Year: 'yyyy",
+                                    "id": "aFG456"
+                                },
+                                {
+                                    "type": "wms",
+                                    "name": "DEA GeoMAD (Landsat 5 TM) v4",
```
Comment (robbibt) [inline dev/terria/terria-cube-v8.json:167]:
```suggestion
                                    "name": "DEA GeoMAD (Landsat 5 TM, version 4)",
```

File: dev/services/wms/ows_refactored/baseline_satellite_data/landsat_annual/ows_landsat_annual_cfg.py [hunk 4]
```diff
@@ -8,6 +8,121 @@
     "title": "DEA Surface Reflectance Calendar Year (Landsat)",
     "abstract": "",
     "layers": [
+        {
+            "title": "DEA GeoMAD (Landsat 8 & 9 OLI-TIRS)",
+            "name": "ga_ls8cls9c_gm_cyear_3",
+            "abstract": """DEA Geometric Median and Median Absolute Deviation (Landsat 8 & 9 OLI-TIRS)
```
Comment (robbibt) [inline dev/services/wms/ows_refactored/baseline_satellite_data/landsat_annual/ows_landsat_annual_cfg.py:14]: Should this be the official expanded product name? E.g. something like

`Geoscience Australia Landsat 8 and 9 Geometric Median and Median Absolute Deviation Collection 3`

File: dev/services/wms/ows_refactored/baseline_satellite_data/landsat_annual/ows_landsat_annual_cfg.py [hunk 5]
```diff
@@ -8,6 +8,121 @@
...
+                "extent_mask_func": "datacube_ows.ogc_utils.mask_by_val",
+                "always_fetch_bands": [],
+                "manual_merge": False,
+            },
+            "styling": {"default_style": "simple_rgb", "styles": styles_c3_ls_list},
+        },
+        {
+            "title": "DEA GeoMAD (Landsat 7 ETM+)",
+            "name": "ga_ls7e_gm_cyear_3",
+            "abstract": """DEA Geometric Median and Median Absolute Deviation (Landsat 7 ETM+)
```
Comment (robbibt) [inline dev/services/wms/ows_refactored/baseline_satellite_data/landsat_annual/ows_landsat_annual_cfg.py:53]: Same as above, should be something like:

Geoscience Australia Landsat 7 Geometric Median and Median Absolute Deviation Collection 3

File: dev/services/wms/ows_refactored/baseline_satellite_data/landsat_annual/ows_landsat_annual_cfg.py [hunk 6]
```diff
@@ -8,6 +8,121 @@
...
+                "extent_mask_func": "datacube_ows.ogc_utils.mask_by_val",
+                "always_fetch_bands": [],
+                "manual_merge": False,
+            },
+            "styling": {"default_style": "simple_rgb", "styles": styles_c3_ls_list},
+        },
+        {
+            "title": "DEA GeoMAD (Landsat 5 TM)",
+            "name": "ga_ls5t_gm_cyear_3",
+            "abstract": """DEA Geometric Median and Median Absolute Deviation (Landsat 5 TM)
```
Comment (robbibt) [inline dev/services/wms/ows_refactored/baseline_satellite_data/landsat_annual/ows_landsat_annual_cfg.py:91]: Geoscience Australia Landsat 5 Geometric Median and Median Absolute Deviation Collection 3?

File: dev/services/wms/ows_refactored/baseline_satellite_data/landsat_annual/ows_landsat_annual_cfg.py (hunk 4 above)
Comment (robbibt) [reply dev/services/wms/ows_refactored/baseline_satellite_data/landsat_annual/ows_landsat_annual_cfg.py:14]: e.g. at the top here:
https://knowledge.dea.ga.gov.au/data/product/dea-geometric-median-and-median-absolute-deviation-landsat/?tab=overview

**Review by SpacemanPaul — APPROVED** [review]
LGTM.

File: dev/services/wms/ows_refactored/baseline_satellite_data/landsat_annual/ows_landsat_annual_cfg.py (hunk 4 above)
Comment (JM-GA) [reply dev/services/wms/ows_refactored/baseline_satellite_data/landsat_annual/ows_landsat_annual_cfg.py]: Thanks Robbi - sounds good to me - abstract has been updated to full expanded name

File: dev/terria/terria-cube-v8.json (hunk 1 above)
Comment (JM-GA) [reply dev/terria/terria-cube-v8.json]: @robbibt- yep version label will be dropped when the existing version is decommissioned - so only one will remain with no versioning in name

**Pull request merged**